SourceX/dx.cpp

```diff
@@ -249,6 +250,77 @@ void LimitFrameRate()
 	frameDeadline = tc + v + refreshDelay;
 }
 
+#ifdef USE_SDL1
+
+void NewHWCursor(SDL_Surface *surf)
+{
+}
+
+#else
+
+static SDL_Surface *cursor_surface = NULL, *cursor_realsurface = NULL;
+static SDL_Cursor *cursor = NULL;
+static int cursor_width, cursor_height;
+static int cursor_surface_palette_version = -1;
+static int cursow_window_res_version = -1;
+extern BOOL oar;

```

Comment:
Use `bool` instead of `BOOL`.

Reply:
The original variable is declared as BOOL. I don't feel like refactoring someone else's code.

Reply:
> someone else's code

There is no such thing. It is our code now.

The idea is that all new booleans use the native `bool` type. If this conflicts with your code somehow, consider refactoring those usages to `bool` too (preferably in a separate change).

Reply:
I should probably correct this my self since i wrote that code.

Reply:
![image](https://user-images.githubusercontent.com/14297035/103465176-a3b5b000-4d39-11eb-8c18-47f04e34b998.png)